### PR TITLE
Added Avatars and Badges

### PR DESCRIPTION
**Pull request by haarsh24**
1.Added Round Avatars 
2.Added Square Avatars
3. Added Badges on avatars
4. Added badges on icons and buttons

### CODE REVIEW

**Review by xii-vi — APPROVED** [review]
Everything is in place, you can merge.

File: components/card.css [hunk 1]
```diff
@@ -88,20 +84,20 @@
...
-  .badges {
+  .card-badges {
     padding-top: 2rem;
     position: relative;
   }
   .btn-badge {
     position: relative;
   }
-  .badge {
+  .card-badge {
```
Comment (viru2001) [inline components/card.css:94]: `card-badges` and `card-badge` looks similar. you can give proper name to this

**Review by viru2001 — COMMENTED** [review]
I suggested minor changes which you can do. Also, try to make utility classes for properties which are repeated. Rest everything is good.

File: components/card.css (hunk 1 above)
Comment (haarsh24) [reply components/card.css:94]: sure

**Pull request merged**